modules/nw-multi-network-policy-ipv6-suppport.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-multi-network-policy-ipv6-support_{context}"]
+= Supporting multi-network policies in IPv6 networks
+
+The ICMPv6 Neighbor Discovery Protocol (NDP) plays a crucial role in IPv6 networks, facilitating the interaction between devices on the same link. NDP is a set of messages and processes that enable devices to discover and maintain information about neighboring nodes.
+
+The Cluster Network Operator (CNO) deploys the iptables implementation of multi-network policy when `useMultiNetworkPolicy` is set to `true`.
+
+To support multi-network policies in IPv6 networks the Cluster Network Operator deploys the following set of rules in every pod affected by a multi-network policy.
+
```

Comment:
I wonder if this YAML example could benefit from a title?

.Example YAML name

That said, other YAML examples do not have names on this page, AFAICT.